### PR TITLE
Remove deprecated global variables

### PR DESCRIPTION
**Pull request by ChristophWurst**
A few apps might break because nobody cares to adapt. So let's remove this sooner than later :)

### CODE REVIEW

**Review by skjnldsv — APPROVED** [review]

**Review by juliusknorr — CHANGES_REQUESTED** [review]
We should at least fix our internal usages in server before removing them. This breaks the files app out of the box :wink: 

```
Uncaught (in promise) ReferenceError: humanFileSize is not defined
    at FileList._createRow (filelist.js:1609)
    at FileList.fileList._createRow (tagsplugin.js:236)
    at FileList.e._createRow (comments.js:1)
    at FileList.e._createRow (additionalScripts.js:1)
    at FileList._renderRow (filelist.js:1793)
    at FileList._nextPage (filelist.js:1281)
    at FileList.setFiles (filelist.js:1359)
    at FileList.reloadCallback (filelist.js:2098)
    at Object.<anonymous> (main.js:25)
    at c (main.js:25)
```

**Comment by ChristophWurst** [comment]
[The files app? Is that thing still around](https://www.youtube.com/watch?v=Q9A0Vufw3NQ)

Yeah, guess we need that beautifully rewritten Vue files app. Then we can easily import the function there :grin:

**Comment by ChristophWurst** [comment]
Oh we can just use `OC.Util.humanFileSize`

File: apps/files/js/filelist.js [hunk 1]
```diff
@@ -3238,7 +3238,7 @@
 			}
 			else {
 				this.$el.find('.selectedActions').removeClass('hidden');
-				this.$el.find('#headerSize a>span:first').text(OC.Util.humanFileSize(summary.totalSize));
+				this.$el.find('#headerSize a>span:first').text(OC.Util.OC.Util.humanFileSize(summary.totalSize));
```
Comment (skjnldsv) [inline apps/files/js/filelist.js]:
```suggestion
				this.$el.find('#headerSize a>span:first').text(OC.Util.humanFileSize(summary.totalSize));
```

File: apps/files/js/filelist.js [hunk 2]
```diff
@@ -2538,7 +2538,7 @@
 							var oldSize = oldFile.data('size');
 							var newSize = oldSize + newFile.data('size');
 							oldFile.data('size', newSize);
-							oldFile.find('td.filesize').text(OC.Util.humanFileSize(newSize));
+							oldFile.find('td.filesize').text(OC.Util.OC.Util.humanFileSize(newSize));
```
Comment (skjnldsv) [inline apps/files/js/filelist.js]:
```suggestion
							oldFile.find('td.filesize').text(OC.Util.humanFileSize(newSize));
```

File: apps/files/js/filelist.js [hunk 3]
```diff
@@ -2398,7 +2398,7 @@
...
```
Comment (skjnldsv) [inline apps/files/js/filelist.js]:
```suggestion
							oldFile.find('td.filesize').text(OC.Util.humanFileSize(newSize));
```

**Comment by ChristophWurst** [comment]
This removes ~100KB from each bundle, btw

**Comment by gary-kim** [comment]
Needs a rebase. :smile:
image
**Continuous** integration?

**Comment by gary-kim** [comment]
There seems to be a JSUnit failure, is that unrelated?
```
PhantomJS 2.1.1 (Linux 0.0.0): Executed 847 of 851 (42 FAILED) (skipped 4) (27.61 secs / 26.648 secs) 
```

**Comment by ChristophWurst** [comment]
Whoopsie. A few tests are indeed failing.

**Comment by ChristophWurst** [comment]
`escapeHTML` is still used in a few places :cry: 

But the rest should be fine.

**Review by gary-kim — APPROVED** [review]
Looks good to me

**Pull request merged**